guide/payments/overview.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: guide
+title: Transaction overview
+description: Principles to strive for while building bitcoin applications for end-users.
+nav_order: 4
+parent: Payments
+permalink: /guide/payments/overview/
+main_classes: -no-top-padding
+image: /assets/images/guide/payments/overview/page-overview.svg
+---
+
+# Transactions overview
+
+Bitcoin is a peer-to-peer push-payment system. This means that you are able to send, or *push*, bitcoin to any address at your will, at any time, without passing through a third party.
+
+
+This is radically different from the legacy financial system where it is often possible for others to *pull* and withdraw money from your account (utility companies, financial institutions, merchants etc.). When you make a payment it will pass through systems that might delay, control or block the payment. 
+## Bitcoin transactions
+
+The most common bitcoin transactions simply contain instructions to move funds from one address to another. Other types of transactions include creating multi-sig wallets or recording data on the blockchain.
+
+Once created, a transaction needs to be signed with the private key(s) of the sender(s), submitted to a node on the network for validation and propagation to other nodes. It then waits to be verified and included in a newly mined block, after which point it will be recorded on the blockchain. For this service, the sender will pay a fee for every transaction which goes to the node who mined the block as a reward.
```

Comment:
```suggestion
Once created, a transaction needs to be signed with the private key(s) of the sender(s), submitted to a node on the network for validation and propagation to other nodes. It then waits to be verified and included in a newly mined block, after which point it will be recorded on the blockchain. For this service, the sender will pay a fee for every transaction which goes to the node that mined the block as a reward.
```
BTW Adding plurals (s) is technically correct but may be confusing since it's been dragging across the entire page.  I didn't edit it out but maybe we can discuss if it's confusing or not?